suricata.yaml.in

```diff
@@ -1839,6 +1839,32 @@ napatech:
     #
     hashmode: hash5tuplesorted
 
+# DPDK configuration
+dpdk:
+    # use RTE-ACL for preflitering SRC-IP,DST-IP,SRC-PORT,DST-PORT
+    pre-acl: yes
+    #use RTE-ACL for FWD - applicable to IPS
+    post-acl: no
+    # allow packet fragementation
+    tx-fragment: no
+    # allow reassembly
+    rx-reassemble: no
+    # mode of operation: BYPASS, IDS, IPS
+    mode: IPS
+    # port index mapping for IPS: [in-port:out-port]
```

Comment:
nit: your example doesn't include a `:`

Reply:
is the correction intended to be formatted from `mode of operation: BYPASS, IDS, IPS` to `to select mode of operation as " mode: BYPASS or mode: IDS mode: IPS"` ?